proto/meta.proto

```diff
@@ -461,6 +462,22 @@ message RescheduleResponse {
   uint64 revision = 2;
 }
 
+message TableParallelism {
+  message FixedParallelism {
+    uint32 parallelism = 1;
+  }
+
+  message AutoParallelism {}
+
+  message CustomParallelism {}
+
+  oneof parallelism {
+    FixedParallelism fixed = 1;
+    AutoParallelism auto = 2;
```

Comment:
Would you mind adding some comments to describe them? For example, the respective behavior when auto scaling is on/off.